examples/dt/uni02beta/control-plane.md

```diff
@@ -0,0 +1,55 @@
+# Configuring networking and deploy the OpenStack control plane
+
+## Assumptions
+
+- A storage class called `local-storage` should already exist.
```

Comment:
I think this statement is not in sync with the storageClass stated in the values.yaml